packages/libs/eda/src/lib/core/types/variable.ts

```diff
@@ -16,3 +16,31 @@ export const StringVariableValue = t.intersection([
     value: t.string,
   }),
 ]);
+
+const _VariableCollectionBase = t.type({
+  entityId: t.string,
+  collectionId: t.string,
+});
+
+export type VariableCollectionDescriptor = t.TypeOf<
+  typeof VariableCollectionDescriptor
+>;
+export const VariableCollectionDescriptor = _VariableCollectionBase;
+
+export function isVariableDescriptor(
```

Comment:
Would this function and `isVariableCollectionDescriptor` be better suited in`utils/study-metadata`?